cocotb_build_libs.py

```diff
@@ -284,7 +272,7 @@ def _get_vpi_lib_ext(
             os.path.join(share_lib_dir, "vpi", "VpiImpl.cpp"),
             os.path.join(share_lib_dir, "vpi", "VpiCbHdl.cpp"),
         ],
-        extra_link_args=_extra_link_args(rpath="$ORIGIN"),
+        extra_link_args=_extra_link_args(lib_name=lib_name, rpath="$ORIGIN"),
```

Comment:
Should we do the same for VHPI and FLI? Even if there is not simulator that supports this on Mac?

Reply:
Sure, I can make them all consistent.